src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -7,8 +7,8 @@ extern SX127xDriver Radio;
 extern SX1280Driver Radio;
 #endif
 
-#ifdef TARGET_R9M_TX
-extern R9DAC R9DAC;
+#if defined(TARGET_R9M_TX) || defined(TARGET_TX_ES915TX)
+extern DAC DAC;
```

Comment:
Change variable name different than class name. This is not good habbit!

Reply:
> Change variable name different than class name. This is not good habbit!

I agree. This was a batch "find and replace", so I didn't pick it up.

Reply:
my fault